client/src/app/videos/+video-watch/video-watch-playlist.component.html

```diff
@@ -14,6 +14,26 @@
         <span>{{ currentPlaylistPosition }}</span><span>{{ playlistPagination.totalItems }}</span>
       </div>
     </div>
+
+    <div class="playlist-controls">
+      <my-global-icon
```

Comment:
IMO the switch with a "auto play playlist" would be better

Reply:
I strongly disagree. The "playlist-controls" area is bound to have multiple buttons like this one: loop, autoplay, random, etc. Having 3 switches will clutter the space, whereas with icons like here, the area stays lean.

Reply:
I would just keep the switch for the auto play, and the icons for the other features. But as you want, I don't have a strong opinion here